inference-engine/src/mkldnn_plugin/nodes/mkldnn_rnn.cpp

```diff
@@ -704,7 +712,40 @@ void MKLDNNRNN::createDescriptor(const std::vector<MemoryDescPtr> &inputDesc,
 }
 
 void MKLDNNRNN::createPrimitive() {
-    auto pd = descs[0].createPrimitiveDescriptorIterator(getEngine());
+    auto engine = getEngine();
```

Comment:
Can we instead use createPrimitiveDescriptor() as it is done for mkldnn_deconv_node.cpp?
It seems in that case it will require less code.

Reply:
createPrimitiveDescriptor() uses descsCompatible() methods internally for Src and Dst descriptors, but for Sequence mode uses swap of first and second dimensions for op->get_input_shape and op->get_output_shape. That is why descriptors is incompatible in createPrimitiveDescriptor()

Reply:
Discussed

Reply:
Done changes according discussion